presentation/src/main/java/org/sagebionetworks/research/presentation/perform_task/PerformTaskViewModel.java

```diff
@@ -399,6 +413,12 @@ public void onChanged(@Nullable final TaskResult taskResult) {
                 compositeDisposable.add(
                         taskResultManagerConnectionSingle
                                 .subscribe((resultManagerConnection) -> {
+                                    resultManagerConnection.addAsyncActionResult(Maybe.fromCallable(
```

Comment:
Are these placeholders? Wondering why they are AsyncActionResults instead of Results

Reply:
I wanted them in the async results since they aren't a step that the user was displayed so I figured the step history was the wrong place for them.